ksql-functional-tests/src/test/resources/query-validation-tests/timestampformat.json

```
@@ -37,6 +37,38 @@
       "outputs": [
         {"topic": "TS", "value": {"ETS": 1566912669200}, "timestamp": 1566912669200}
       ]
+    },
+    {
+      "name": "KSQL override output timestamp for CSAS",
+      "statements": [
+        "CREATE STREAM TEST (ID bigint, EVENT_TS bigint) WITH (kafka_topic='test_topic', value_format='JSON', timestamp='EVENT_TS');",
```

The test case isn't testing `timestamp` in the with clause of the CS statement, so remove it, as it may confuse people.  Which means you can remove `EVENT_TS` too.

```suggestion
        "CREATE STREAM TEST (ID bigint) WITH (kafka_topic='test_topic', value_format='JSON');",
```

---

Done